### PR TITLE
UI elements

### PR DESCRIPTION
**Pull request by animeshgarai09**
I just finished the home page design for social media. this one is a reasonably simple PR. I need help finding typos or any CSS mistakes for just now.

### CODE REVIEW

File: src/components/create-post/create-post.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import styles from "./create-post.module.scss"
+import { useCallback, useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+// import { BsFileEarmarkImage } from "@icons"
+import { Modal, ModalBody, useToast } from "react-felix-ui"
+import image from "@assets/images/picture.png"
+// import ImageCrop from "../image-crop/image-crop"
+// import getCroppedImg from "@global/js/cropImage"
+// import WritePost from "../write-post/write-post"
```
Comment (Developers-codz) [inline src/components/create-post/create-post.js:9]: remove commented code

File: src/components/create-post/create-post.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    // const handleNext = (images) => {
+    //     setNext(true)
+    //     const promises = images.map(async ({ image, croppedArea }) => {
+    //         return await getCroppedImg(image, croppedArea)
+    //     })
+
+    //     Promise.all(promises).then((croppedImages) => {
+    //         setCroppedImages(croppedImages)
+    //     })
+    // }
```
Comment (Developers-codz) [inline src/components/create-post/create-post.js:28]: here also if its not needed later

File: src/components/create-post/create-post.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                isOpen={isModalOpen}
+                size={isNext ? "xxl" : "md"}
+                onClose={handleModalClose}
+                title={isNext ? "Create new post" : "Crop Images"}
+                closeOnOverlayClick={false}
+                headerClassName={styles.modal_header}
+            >
+                <ModalBody className={styles.modal_body}>
+                    {/* <ImageCrop isHidden={isNext} selectedImages={droppedImages} handleNext={handleNext} />
+                    {isNext && croppedImages && <WritePost images={croppedImages} onBack={() => setNext(false)} />} */}
```
Comment (Developers-codz) [inline src/components/create-post/create-post.js:83]: here as well

File: src/components/image-viewer/image-viewer.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                }
+                break
+            case "prev":
+                if (currentIndex !== 0) {
+                    setCurrentIndex(prev => prev - 1)
+                }
+                break
+            default:
+                break
+        }
```
Comment (Developers-codz) [inline src/components/image-viewer/image-viewer.js:23]: Appreciation - for proper handling using switch case

File: src/components/post-card/post-card.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                </div>
+                <div className={styles.options}>
+                    <IconButton
+                        icon={<MdBookmarkBorder />}
+                        // onClick={() => handleSlide("next")}
+                        className={styles.icon}
+                    />
+                    <IconButton
+                        icon={<RiShareForwardLine />}
+                        // onClick={() => handleSlide("next")}
```
Comment (Developers-codz) [inline src/components/post-card/post-card.js:48]: remove commented code from here also and on line 24 and 43

File: src/components/right-panel/right-panel.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
+                    <div className={styles.follower_card}>
+                        <Avatar name="Animesh garai" size="sm" className={styles.avatar} />
+                        <span>Animesh Garai</span>
+                    </div>
```
Comment (Developers-codz) [inline src/components/right-panel/right-panel.js:81]: can be done later Most of the JSX is repeating you can create array of object and map over it

**Review by Developers-codz — APPROVED** [review]
UI is amazing as always. Also Good work on code
There is horizontal scroll although in the UI

File: src/components/create-post/create-post.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+    return (
+        <>
+            <div className={styles.dropZone} {...getRootProps()}>
+                <div className={`${styles.inner} ${isDragActive && styles.border}`} >
+                    <input {...getInputProps()} />
+                    {/* <Avatar name="Animesh garai" size="sm" className={styles.avatar} /> */}
+                    <img src={image} alt="" />
+                    {/* <BsFileEarmarkImage className={styles.icon} /> */}
+                    {
```
Comment (vishal804) [inline src/components/create-post/create-post.js:65]: commented code can be removed if not needed

**Review by vishal804 — APPROVED** [review]
Everything Looks Good 
UI is Superb   Nice Work

**Pull request merged**